config/schedule.rb

```diff
@@ -1,3 +1,6 @@
+rails = File.join Dir.getwd, "config", "environment.rb"
+require rails
```

Comment:
The Rails environment doesn't appear to be loaded when building the crontab so I added this. Really I just needed `url_helpers` so I'm unsure if this is overkill.